src/NuGet.Core/NuGet.Packaging/Exceptions/MinClientVersionException.cs

```diff
@@ -1,15 +1,15 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
 using NuGet.Common;
-using NuGet.Packaging.Core;
 
 namespace NuGet.Packaging
 {
     /// <summary>
     /// Custom exception type for a package that has a higher minClientVersion than the current client.
     /// </summary>
-    public class MinClientVersionException : PackagingException, ILogMessageException
+    public class MinClientVersionException : Exception, ILogMessageException
```

Comment:
Why did this change? I'm worried this will break other things. This should stay a PackagingException if possible

Reply:
I did this to use the interface instead of making PackagingException a base class. Here is hte error if I change it back. MinClientVersionException.AsLogmessage(): cannot override inherited member 'PackagingException.AsLogMessage(0 because it is not marked virutal abstract or override.

Reply:
If you want me to make PackagingException a base class or add another interface IPackagingException here I can do that, or we can leave it like this.

Reply:
I think using `PackagingException` as a base class and making AsLogMessage virtual would be best unless you know of a good reason to avoid this.